config/routes.rb

```diff
@@ -1,21 +1,21 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  devise_for :users, controllers: {
+  devise_for :users, skip: :registrations, controllers: {
     sessions: "users/sessions",
   }
 
   devise_scope :user do
-    get "email_confirmation", to: "users/sessions#redirect_from_magic_link"
-    post "sign_in_with_token", to: "users/sessions#sign_in_with_token"
-
-    namespace :induction_coordinator do
-      namespace :registration do
-        get "school_confirmation", to: "/users/induction_coordinators/registrations#confirm_school"
-        get "check_email", to: "/users/induction_coordinators/registrations#start_registration"
-        post "check_email", to: "/users/induction_coordinators/registrations#check_registration_email"
-        get "register", to: "/users/induction_coordinators/registrations#new"
-        post "register", to: "/users/induction_coordinators/registrations#create"
+    get "/users/confirm_sign_in", to: "users/sessions#redirect_from_magic_link"
+    post "/users/sign_in_with_token", to: "users/sessions#sign_in_with_token"
+
+    namespace :induction_coordinators do
```

Comment:
Do we still want this `induction_coordinators` namespace?

Reply:
I think we'll need it as the project progresses but perhaps not for this. Happy to take it out.

Reply:
I thought it would make sense for it to mirror the file structure, but I don't have strong feelings about it. Whatever you think is best 🙂